tests/Command/GenerateObjectAclCommandTest.php

```diff
@@ -51,7 +51,7 @@ public function testExecuteWithDeprecatedDoctrineService(): void
         $application = new Application();
         $application->add($command);
 
-        $command = $application->find(GenerateObjectAclCommand::getDefaultName());
+        $command = $application->find('sonata:admin:generate-object-acl');
```

Comment:
you can use the static property too

Reply:
The static property is protected.
Here the error is that `getDefaultName()` return type is string|null.

I don't think it's bad to copy paste the name of the command.
This way if someone rename the command, test will fail, and it will avoid a regression.

Reply:
After doing a similar change on sonata media, I think it is better to just use the static property,can you take a look on how I did it?

Reply:
Can you provide me the link ?

Reply:
nvm this is the test! I thought it was the command class